src/sympc/module/nn/conv.py

```diff
@@ -57,6 +57,7 @@ def __init__(self, session: Session) -> None:
         self.padding = 0
         self.dilation = 1
         self.groups = 1
+        self._parameters = None
```

Comment:
Q: Could you add those to slots and change the slots to a tuple from the list?

Reply:
Yeah sure.

Reply:
Done